src/plugin/plugin.rs

```diff
@@ -214,6 +234,17 @@ where
                     Self::get_systems(PhysicsSet::Writeback).in_set(PhysicsSet::Writeback),
                 ),
             );
+
+            // Warn user if the timestep mode isn't in Fixed
+            if self.schedule.as_dyn_eq().dyn_eq(FixedUpdate.as_dyn_eq()) {
+                let config = app.world.resource::<RapierConfiguration>();
+                match config.timestep_mode {
+                    TimestepMode::Fixed { .. } => {}
+                    mode => {
+                        warn!("TimestepMode is set to `{:?}`, it is recommended to use `TimestepMode::Fixed` if you have the physics in `FixedUpdate`", mode);
+                    }
+                }
+            }
```

Comment:
Should we automatically set it to fixed here?

Reply:
I’m fine with the warning for now.